src/libs/actions/Report.js

```diff
@@ -1638,23 +1638,28 @@ function shouldShowReportActionNotification(reportID, action = null, isRemote =
 
 /**
  * @param {String} reportID
- * @param {Object} action
+ * @param {Object} reportAction
  */
-function showReportActionNotification(reportID, action) {
-    if (!shouldShowReportActionNotification(reportID, action)) {
+function showReportActionNotification(reportID, reportAction) {
+    if (!shouldShowReportActionNotification(reportID, reportAction)) {
         return;
     }
 
     Log.info('[LocalNotification] Creating notification');
-    LocalNotification.showCommentNotification({
-        report: allReports[reportID],
-        reportAction: action,
-        onClick: () => {
-            // Navigate to this report onClick
-            Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(reportID));
-        },
-    });
-    notifyNewAction(reportID, action.actorAccountID, action.reportActionID);
+    const report = allReports[reportID];
+
+    const notificationParams = {
+        report,
+        reportAction,
+        onClick: () => Navigation.navigate(ROUTES.getReportRoute(reportID)),
+    };
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/28328. We should have kept using `ROUTES.REPORT_WITH_ID.getRoute` because `ROUTES.getReportRoute` no longer exists.